src/core/routing/transfer-routes/multi-transfer.ts

```
@@ -0,0 +1,415 @@
+        );
+
+        // Assign priority based on destination presence - LOWER values = HIGHER priority
+        let basePriority = 1000; // Start with a high value (low priority)
+
+        if (directStopsToDestination >= 0) {
+          // Direct line to destination - highest priority
+          // Lower number for fewer stops = higher priority
+          basePriority = 100 + directStopsToDestination;
+        } else {
+          // Get the minimum transfers from this line to any destination line
+          let minTransfersToDestination = Infinity;
```

Biome (1.9.4)

[error] Use Number.POSITIVE_INFINITY instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.POSITIVE_INFINITY instead.


(lint/style/useNumberNamespace)